### ISSUE TITLE
Feature Request: Implement Project Search by Used Technology or Name

### ISSUE THREAD

**Issue by MrAshwin2142** [MrAshwin2142/The-Wall-of-Projects#63]
#### Description
The current setup lacks a search functionality to find projects based on their used technology or project name. Incorporating a search feature will significantly improve user navigation and accessibility within the repository.

#### Proposed Solution
- Introduce a search bar or filter mechanism enabling users to search projects by used technology or project name.
- Enhancing user experience by allowing quick and specific searches within the repository.

#### Expected Benefits
- Improved accessibility for users to find projects matching their interests.
- Enhances user interaction and exploration within the repository.

#### Additional Information
The addition of a search feature will enhance user interaction and streamline the process of discovering projects based on specific technologies or project names.

**Comment by animasoul** [comment]
hi @MrAshwin2142, let me take this on.

- add a data-tech attribute to each button
- only display the buttons based on the search term

**Comment by MrAshwin2142** [comment]
good idea @animasoul , go ahead ..

**Comment by animasoul** [comment]
created pull request.. added search by name & contributor.

It's quite a simple one, maybe someone else would like to improve? Styling and maybe predictive based on the data attributes listed?

**Comment by MrAshwin2142** [comment]
Right, @animasoul . That's why I am not closing this issue; we can further improve this.

**Comment by Victorthedev** [comment]
Hello, I will like to be assigned so i can try to improve it

**Comment by MrAshwin2142** [comment]
@Victorthedev any updates??

**Comment by Victorthedev** [comment]
@MrAshwin2142 Yes!, created a pull request

**Comment by Victorthedev** [comment]
done

**Comment by jaiswalsachin49** [comment]
Is this still open?
